package.json

```diff
@@ -29,7 +29,8 @@
     "fivebeans": "^1.3.1",
     "is-async-func": "^1.0.0",
     "lodash": "^4.6.1",
-    "moment": "^2.12.0"
+    "moment": "^2.12.0",
+    "node-uuid": "^1.4.8"
```

Comment:
This package has been deprecated
https://www.npmjs.com/package/node-uuid
Please use uuid instead